almanac.lic

```diff
@@ -20,6 +22,14 @@ class Almanac
     return if XMLData.room_title.include? 'Carousel Chamber'
     return if hidden?
 
+    if @almanac_skills
+      training_skill = @almanac_skills
+                       .select { |skill| @almanac_skills }
```

Comment:
What does this line do?

Reply:
I don't know besides it working, was a cut and paste from another place :) Is there a more proper way to test all the skills in almanac_skills: for < 18 mindstates and use that skill?